traitlets/config/tests/test_configurable.py

```diff
@@ -368,7 +368,7 @@ class SomeSingleton(SingletonConfigurable):
             pass
 
         class DefaultConfigurable(Configurable):
-            a = Integer(config=True)
+            a = Integer().tag(config=True)
             def _config_default(self):
                 if SomeSingleton.initialized():
                     return SomeSingleton.instance().config
```

Comment:
Okay, then https://github.com/jasongrout/traitlets/commit/eab41d651c828a7f62131adb589f91c32d2336b5 should be sufficient (i.e., the commit below).